<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderLogPropertiesDialog</class>
 <widget class="QWidget" name="OrderLogPropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>176</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Order Log</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Resources/Resources.qrc">
    <normaloff>:/icons/window_earth.png</normaloff>:/icons/window_earth.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="m_orderStatusLayout">
     <item>
      <spacer name="m_leftSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="m_orderStatusGroup">
       <property name="title">
        <string>Order Status Filter</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QRadioButton" name="m_allOrderStatusButton">
          <property name="text">
           <string>All Orders</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_liveOrderStatusButton">
          <property name="text">
           <string>Live Orders</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_terminalOrderStatusButton">
          <property name="text">
           <string>Terminal Orders</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_customOrderStatusButton">
          <property name="text">
           <string>Custom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="m_orderStatusSeparator">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="m_rightSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="m_buttonsLayout">
     <item>
      <widget class="QPushButton" name="m_okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_applyAllButton">
       <property name="text">
        <string>Apply All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
